serving/src/main/java/feast/serving/config/FeastProperties.java

```diff
@@ -316,6 +317,13 @@ public BigTableStoreConfig getBigtableConfig() {
       return new BigTableStoreConfig(this.config.get("project_id"), this.config.get("instance_id"));
     }
 
+    public CassandraStoreConfig getCassandraConfig() {
+      return new CassandraStoreConfig(
```

Comment:
We will need something like a connection string, because we can potentially give more than one node address